iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/variant.inl

```diff
@@ -130,7 +140,7 @@ void variant<Types...>::call_element_destructor() noexcept
 template <typename... Types>
 template <typename T>
 inline typename std::enable_if<!std::is_same<T, variant<Types...>&>::value, variant<Types...>>::type&
-variant<Types...>::operator=(T&& f_rhs) noexcept
+variant<Types...>::operator=(T&& rhs) noexcept
```

Comment:
While I appreciate the removal of the `f` prefix, it adds quite some noise if it was used that massively like here. Please consider to do it in a separate PR it there are so many occurrences.

Reply:
[Mimimimimi](https://youtu.be/_goMQolXcbs?t=19) :stuck_out_tongue: 

I'll consider it next time, but remember being a reviewer is not always a picnic :upside_down_face: